lib/zendesk_api/error.rb

```diff
@@ -20,7 +20,7 @@ def initialize(*)
         super
 
         if response[:body].is_a?(Hash)
-          @errors = response[:body]["details"] || generate_error_msg(response[:body])
+          @errors = response[:body]["details"] || generate_error_msg(response[:body]) || response[:body]["error"]
```

Comment:
There look to be a few different patterns for errors that could come up in addition to this (eg. `errors` is the key returned in a validation message when attempting to create a translation for a locale that is not supported)

Since this all is returned in the `response[:body]`, it feels like a better fit to consolidate all error message handling in `generate_error_msg` instead of doing some bits here, some there (but not blocking, possibly something to consider - or save for a followup).